src/_product-updates/2025-09-29-documentation-enhancements-and-fixes.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Documentation enhancements and fixes
+tags: [Improvement]
+---
+While achieving milestones on larger projects, we’ve also made many quality-of-life enhancements and bug fixes. Here's the changelog:
+
+Enhancements:
+- [Custom meta tags](/help/specification-support/meta-tags/) can now be added to the head of your documentation,
+- Dead anchors now redirect to their parent operation instead of the top of the documentation,
+- In Embed mode, the search placeholders now specify their context (« Search API documentation » vs « Search hub »), so users can differentiate the global search from the documentaion portal search, 
+- A confirmation modal has been added to the API Explorer before sending destructive requests to avoid unwanted losses,
+- The scrolling experience has been globally reworked to ensure its stability,
+- When using groups to categorize your API documentation inside your hub, you now have the option to only display the ones from the same category in the documentation selector of a documentation,
+- The browser window.title is now automatically updated when scrolling between operations to display the current navigation context,
+- Highlights now work for subtitles inside x-topics, and global/operation descriptions,
+- [x-feedbackLink](/help/publish-documentation/feedback/) now works with Swagger 2.0 documentation,
+- Groups in the navigation bar are now collapsible, to facilitate the navigation between groups containing many operations,
+- The "add custom domain" module has been reworked to provide better explanations and error messages,
+- Branches can now be [sorted alphabetically in ascending and descending order](/help/publish-documentation/branching/#sort-branches),
+- Skeletons for previews have been enhanced,
+- x-state badges can now have [custom colors](/help/specification-support/doc-badges/#custom-color),
+- Accordions can now be added in Markdown using `<details>` and `<summary>` tags,
```

Comment:
We should document this under https://docs.bump.sh/help/documentation-experience/markdown-support/